src/lib/kdump/clients/kdump.rb

```diff
@@ -80,7 +80,7 @@ def main
             "handler" => fun_ref(method(:cmdKdumpDumpFormat), "boolean (map)"),
             # TRANSLATORS: CommandLine help
             "help"    => _(
-              "Dump format for dump image none/ELF/compressed/lzo"
+              "Dump format for dump image: none/ELF/compressed/lzo"
```

Comment:
Thanks for patch. As it change string that is translatable, we also need changelog entry ( with some issue number like bugzilla one, github one or jira entry ) and also bump version, otherwise there is risk of desync of translatable strings and used ones.

Reply:
I'm sorry, I don't reported to Bugzilla about it.
I found missed colon when translate yast-kdump to Ukrainian and was create commit.
I need some time for understand how create bugreport on Bugzilla

Reply:
Version bumped and changelog updated. See https://bugzilla.suse.com/show_bug.cgi?id=1195372